src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -771,15 +771,19 @@ static function (ChildDefinition $definition, AsPeriodicTask|AsCronTask $attribu
         $container->registerAttributeForAutoconfiguration(AsMessage::class, static function (ChildDefinition $definition) {
             $definition->addExcludeTag('container.excluded.messenger.message');
         });
-        $container->registerAttributeForAutoconfiguration(Entity::class, static function (ChildDefinition $definition) {
-            $definition->addExcludeTag('container.excluded.doctrine.entity');
-        });
-        $container->registerAttributeForAutoconfiguration(Embeddable::class, static function (ChildDefinition $definition) {
-            $definition->addExcludeTag('container.excluded.doctrine.embeddable');
-        });
-        $container->registerAttributeForAutoconfiguration(MappedSuperclass::class, static function (ChildDefinition $definition) {
-            $definition->addExcludeTag('container.excluded.doctrine.mapped_superclass');
-        });
+
+        // DoctrineBundle autoconfigures attributes since version 2.14.0
+        if (!array_key_exists(Entity::class, $container->getAutoconfiguredAttributes())) {
```

Comment:
I'm not sure this is resilient if the `DoctrineBundle` is registered before `FrameworkBundle`. We can check the package version instead.

```suggestion
        if (InstalledVersions::isInstalled('doctrine/doctrine-bundle') && version_compare(InstalledVersions::getVersion('doctrine/doctrine-bundle'), '2.14.0', '<')) {
```

Reply:
this is not resilient at all irrespective of the order, because DI extensions receive a separate container builder (which is then merged into the main one), precisely to reduce the cases of writing logic that depends on the order (which is not guaranteed at all by Flex)

Reply:
Is the composer version check better?